documentation/src/main/markdown/releasenotes.md

```diff
@@ -1,5 +1,15 @@
 # Release notes
 
+## Version 8.4.0
+
+### New features
+
+### Changed features
+
+* The flag value EPHEMERAL has been deprecated in favor of the value STATELESS.  The behavior of the scan mode is generally unchanged.  See the [Stateless Scans page](downloadingandrunning/statelessscan.md) for further details.
```

Comment:
If there aren't any changes to the behavior, we could omit the word "unchanged" to avoid confusion. If there is any behvaiour change that is noticeable to an end user, could we note the specifics here?

Reply:
I can do that.